README.md

```diff
@@ -77,6 +77,7 @@ ADDITIONAL_TF_OVERRIDE_LOCATIONS=/path/to/module1,path/to/module2 tflocal plan
 
 ## Change Log
 
+* v0.24.0: Add support to return `terraform-local` version when calling `tflocal -version` and fix AWS provider detection
```

Comment:
this doesn't sound clear... lmk if you find something better, I'd happily take any suggestion 😅